timesource/timesource_test.go

```diff
@@ -175,8 +176,50 @@ func TestNTPTimeSource(t *testing.T) {
 				timeQuery:       tc.query,
 			}
 			assert.WithinDuration(t, time.Now(), source.Now(), clockCompareDelta)
-			source.updateOffset()
+			_ = source.updateOffset()
```

Comment:
assert.Nil(t, ...) ?

Reply:
I'll try to adapt the test cases as i think some of them produce an error.